searx/plugins/tracker_url_remover.py

```diff
@@ -39,20 +36,53 @@ def __init__(self, plg_cfg: "PluginCfg") -> None:
             preference_section="privacy",
         )
 
-    def on_result(
-        self, request: "SXNG_Request", search: "SearchWithPlugins", result: Result
-    ) -> bool:  # pylint: disable=unused-argument
-        if not result.parsed_url:
+    def on_result(self, request: "SXNG_Request", search: "SearchWithPlugins", result: Result) -> bool:
+
+        result.filter_urls(self.filter_url_field)
+        return True
```

Comment:
To remove tracker arguments from *all* URLs (including links in the infobox and other areas) we have to use the [`Result.filter_urls(..)`](https://docs.searxng.org/dev/result_types/base_result.html#searx.result_types._base.Result.filter_urls) method / its the same procedure we have in the hostname-replace plugin.